app/controllers/idv/capture_doc_controller.rb

```diff
@@ -22,7 +22,7 @@ def ensure_user_id_in_session
     end
 
     def add_unsafe_eval_to_capture_steps
-      return unless %w[mobile_front_image capture_mobile_back_image].include?(current_step)
+      return unless %w[mobile_front_image capture_mobile_back_image selfie].include?(current_step)
```

Comment:
is there any way we can link this constant to the pages where the JS is actually served? How can we be more proactive about detecting and catching this in the future?

Reply:
It's any view that has this: `render('idv/acuant_sdk_document_capture_form',` but the `acuant_sdk_document_capture_enabled` has to be true also.

Reply:
Ok then how does the selfie view get it?

```
git grep acuant_sdk_document_capture_form -- app  
app/views/idv/capture_doc/capture_mobile_back_image.html.erb:      'idv/acuant_sdk_document_capture_form',
app/views/idv/capture_doc/mobile_front_image.html.erb:      'idv/acuant_sdk_document_capture_form',
app/views/idv/doc_auth/mobile_back_image.html.erb:      'idv/acuant_sdk_document_capture_form',
app/views/idv/doc_auth/mobile_front_image.html.erb:      'idv/acuant_sdk_document_capture_form',
```